File: python/0026.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if not nums: return 0
        i = 1
        j = 0
        while i < len(nums):
            if nums[i] == nums[j]:
                i += 1
                continue
            else:
                j += 1
                nums[j] = nums[i]
                i += 1
        return j + 1

File: rust/src/bin/0026.rs
impl Solution {
    pub fn remove_duplicates(nums: &mut Vec<i32>) -> i32 {
        if nums.len() == 0 {
            return 0;
        }
        let mut i = 1;
        let mut j = 0;
        while i < nums.len() {
            if nums[i] == nums[j] {} else {
                j += 1;
                nums[j] = nums[i];
            }
            i += 1;
        }
        (j + 1) as i32
    }
}